### DISCUSSION TITLE
Notification System Updates

### DISCUSSION THREAD

**Discussion by JasonAGross** [directus/directus#16184] [Feature Requests]
I have made some changes locally to (hopefully) improve upon the notification process as well as address the bug listed here: https://github.com/directus/directus/issues/13727

That PR exists in my own repo for comparison purposes here: https://github.com/JasonAGross/directus/pull/1

I wanted to get feedback from the core team before changing my upstream into the directus repo here and formally opening a PR. I've made some design decisions (both UX and technical) on my own here, trying to stay consistent as much as possible but this is also my first contribution request into this project. 

Some highlights of those decisions: 

- Add a new system field to `directus_notifications` of `read` 
- Update the notification badge in the UI to only count 'unread' items in the inbox, this is a departure from the current count which includes the full inbox
- When a notification is not attached to a collection the user will see the notification message on an additional `v-drawer` when clicking on it. 
- I'm using that additional drawer as a sibling element on the component. I don't see any other examples of using fragments/multiple root elements in the codebase so let me know if I need to resolve that.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
